packages/server/lib/routes.ts

```diff
@@ -187,8 +188,8 @@ publicAPI.route('/config/:providerConfigKey').get(apiAuth, configController.getP
 publicAPI.route('/config').post(apiAuth, configController.createProviderConfig.bind(configController));
 publicAPI.route('/config').put(apiAuth, configController.editProviderConfig.bind(configController));
 publicAPI.route('/config/:providerConfigKey').delete(apiAuth, deletePublicIntegration);
-publicAPI.route('/integrations').get(apiAuth, getPublicListIntegrations);
-publicAPI.route('/integrations/:uniqueKey').get(apiAuth, getPublicIntegration);
+publicAPI.route('/integrations').get(connectSessionOrApiAuth, getPublicListIntegrations);
+publicAPI.route('/integrations/:uniqueKey').get(connectSessionOrApiAuth, getPublicIntegration);
```

Comment:
forgot to say we also need all the `/oauth/ + /auth/ + /api-auth/` routes

Reply:
I will change the auth middleware for auth endpoints in the next PR when I modified the frontend sdk where those endpoints are being called from